core/server/master/src/main/java/alluxio/master/metastore/BlockStore.java

```diff
@@ -24,7 +24,7 @@
  * The block store keeps track of block sizes and block locations.
  */
 @ThreadSafe
-public interface BlockStore extends Iterable<Block> {
```

Comment:
Are there any use-cases where it's important that `BlockStore` itself is an `Iterable`?